crates/red_knot/tests/cli.rs

```diff
@@ -216,9 +216,115 @@ fn rule_severity() -> anyhow::Result<()> {
     })
 }
 
-/// Red Knot warns about unknown rules
+/// The rule severity can be changed using `--ignore`, `--warn`, and `--error`
 #[test]
-fn unknown_rules() -> anyhow::Result<()> {
+fn cli_rule_severity() -> anyhow::Result<()> {
+    let case = TestCase::with_file(
+        "test.py",
+        r#"
+        y = 4 / 0
+
+        for a in range(0, y):
+            x = a
+
+        print(x)  # possibly-unresolved-reference
+        "#,
+    )?;
+
+    case.insta_settings().bind(|| {
+        // Assert that there's a possibly unresolved reference diagnostic
+        // and that division-by-zero has a severity of error by default.
+        assert_cmd_snapshot!(case.command(), @r"
+            success: false
+            exit_code: 1
+            ----- stdout -----
+            error[lint:division-by-zero] <temp_dir>/test.py:2:5 Cannot divide object of type `Literal[4]` by zero
+            warning[lint:possibly-unresolved-reference] <temp_dir>/test.py:7:7 Name `x` used when possibly not defined
+
+            ----- stderr -----
+        ");
+
+
+        assert_cmd_snapshot!(
+            case
+                .command()
+                .arg("--ignore")
+                .arg("possibly-unresolved-reference")
+                .arg("--warn")
+                .arg("division-by-zero")
+                .arg("--warn")
+                .arg("unresolved-import"),
```

Comment:
Why is this included, when there's no unresolved-import diagnostic in the code? Is it just to verify that it's fine to specify a level for a diagnostic that doesn't occur?

Reply:
it's mainly to assert that you can specify multiple arguments. I admit, I was too lazy to write another test (just so much boilerplate)